src/util/directiontables.cpp

```diff
@@ -118,3 +118,8 @@ const u8 wallmounted_to_facedir[6] = {
 	8,
 	4 + 2
 };
+
+const u8 facedir_to_dir[6] = {
```

Comment:
"facedir" here is a bit misleading since it only handles the axis, not really the same as `minetest.facedir_to_dir`
This is why I suggested moving the table inline (it is only used once anyway).

Reply:
Fixed.